testutils/utils.go

```diff
@@ -388,5 +422,7 @@ func CheckSysctlValue(sysctlPath, value string) error {
 		return fmt.Errorf("error asserting sysctl value: expected: %s, got: %s for sysctl path: %s", value, string(buf), sysctlPath)
 	}
 
+	_ = fh.Close()
```

Comment:
what was wrong with using defer?

Reply:
can't do it because Close returns an error which needs to be checked or ignored

Reply:
can we prevent validation in the testutils module since it is just test code?

Reply:
I think there's no easy fix since this is from the go-build container which affects all the Calico repos